XML/Other/ReplaceableTestParameters.xml

```diff
@@ -478,4 +478,8 @@ Scenario 2 : You need to run all the performance tests quickly.
 		<ReplaceThis>EXPECTED_MLNX_VERSION</ReplaceThis>
 		<ReplaceWith>4.5-0.0.8</ReplaceWith>
 	</Parameter>
+	<Parameter>
```

Comment:
Although this code is good, I personally don't like the fact we have to carry this parameter in all test cases, just for the case where we might be on redhat/centos 6.x.

One suggestion would be to hardcode the FIO_VERSION inside the function, so we don't have to change all test cases.

Reply:
Hardcoded the FIO_VERSION in source code block itself